pkg/util/log/e2e/e2e.go

```diff
@@ -0,0 +1,30 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/metrics"
+	"github.com/Azure/ARO-RP/pkg/metrics/statsd/e2e"
+)
+
+type E2EHook struct {
+	logToMetrics e2e.E2ELogToMetrics
+}
+
+func NewE2EHook(m metrics.Emitter) E2EHook {
+	return E2EHook{
+		logToMetrics: e2e.NewE2ELogToMetrics(m),
+	}
+}
+
+func (E2EHook) Levels() []logrus.Level {
+	return logrus.AllLevels
+}
+
+func (h *E2EHook) Fire(entry *logrus.Entry) error {
+	h.logToMetrics.PostMetricsFromLogEntry(entry)
+	return nil
```

Comment:
What's the purpose of returning an `error` here if we're only going to be returning `nil`? Should there be some sort of checking?

Reply:
The function definition must match the logrus `Hook` interface - https://github.com/sirupsen/logrus/blob/master/hooks.go